ci/lint/06_script.sh

```diff
@@ -10,6 +10,8 @@ GIT_HEAD=$(git rev-parse HEAD)
 if [ -n "$CIRRUS_PR" ]; then
   COMMIT_RANGE="${CIRRUS_BASE_SHA}..$GIT_HEAD"
   test/lint/commit-script-check.sh "$COMMIT_RANGE"
+else
+  COMMIT_RANGE="$(git fetch "$CIRRUS_REPO_CLONE_URL" "${CIRRUS_LAST_GREEN_CHANGE:-$CIRRUS_DEFAULT_BRANCH}" && git merge-base FETCH_HEAD HEAD)..$GIT_HEAD"
```

Comment:
My suggestion would be to simply set `COMMIT_RANGE="EMPTY"` and then use that to skip the affected linters

Reply:
https://github.com/bitcoin/bitcoin/pull/26588